bindings/bind_py_module.cc

```diff
@@ -33,10 +33,9 @@ namespace py = pybind11;
 PYBIND11_MODULE(_rascal, mod) {
   mod.doc() = "Python bindings for the Rascal library";
 
-  py::module m_strc_mng = mod.def_submodule("StructureManager");
-  m_strc_mng.doc() = "Structure Manager Classes";
-  py::module m_adp = mod.def_submodule("Adaptor");
-  m_adp.doc() = "Adaptor Classes";
+  py::module m_nl = mod.def_submodule("NeighbourList");
```

Comment:
forgot to change the string. From the python side Adaptors and StructureManagers are not usable anymore because of the weak_pointer. So NeighbourList contains the factories and as submodules Adaptors and StructureManagers but these submodules should not be used directly.

Reply:
I updated this